scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/SimpleUsersConfig.java

```diff
@@ -9,6 +9,19 @@
 
     @Override
     public void configure(SimpleUsers simpleUsers) {
-        
+    /*
+         simpleUsers.emailUsername("");
+         simpleUsers.emailPassword("");
+         simpleUsers.emailServerHost("");
+         simpleUsers.emailServerPort("");
+  		 simpleUsers.emailFrom("");
+  		 simpleUsers.instanceUrl(""); 
+         simpleUsers.appName("");
+     */
```

Comment:
remove if not needed; also, you probably want to put email configurations into a config file and use the configurations here (`@Value`, `@PropertySource`)